docs/src/rest-api.md

```diff
@@ -0,0 +1,96 @@
+# REST API Reference
+
+REST API for programmatic access to the faucet service.
```

Comment:
I would probably expand the intro a bit to explain the flow, or at the very lead link to the explanation of the flow in the architecture section.